src/lib/util.ts

```diff
@@ -5,15 +5,18 @@ import {
   IProposalStage,
   IProposalState,
   IRewardState,
-  ISchemeState } from "@daostack/client";
+  ISchemeState} from "@daostack/client";
 import { GenericSchemeRegistry } from "genericSchemeRegistry";
 import { of } from "rxjs";
 import { catchError } from "rxjs/operators";
 
 import BN = require("bn.js");
+import "moment";
```

Comment:
what does this do?

Reply:
See the timezone date functions I added

Reply:
you did not really answer the question, and I also wonder wha ta "bare"  import does here (it it is because importing "moment" hsa side effects that we want, then we need a comment)

Reply:
/**
 * gotta load moment in order to use moment-timezone directly
 */

Reply:
the competition UI displays datetimesthat include the name of the users' local timezone and their GMT offset